app/Models/Category.php

```diff
@@ -171,15 +171,15 @@ public function itemCount()
 
         switch ($this->category_type) {
             case 'asset':
-                return $this->assets()->count();
+                return $this->assets->count();
```

Comment:
Nice fixes, these! I think these are ones that could really bite us in the tookus. Still - not to sound like a drone - in a separate PR I would've preferred.

Reply:
I would have too, if I could have made those new tests pass. But here we are.